src/main/java/io/zold/api/Taxes.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2018 Yegor Bugayenko
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package io.zold.api;
+
+import org.cactoos.Proc;
+import org.cactoos.text.FormattedText;
+import org.cactoos.text.UncheckedText;
+
+/**
+ * Taxes payment algorithm.
+ *
+ * @since 1.0
+ * @todo #40:30min Implement tax payment to remote nodes.
+ *  Payment should happen only if the wallet is in debt of more than
```

Comment:
@victornoel "Payment should happen only if the wallet is in debt of more than 1 Zold" is not correct. You can definitely pay your taxes before you accumulate 1 Zold of tax debt. Remove this line.

Reply:
@llorllale I know you can, it was a proposal for a first implementation, then maybe it can be customizable. But I can remove the line if you prefer, or reword to be clearer.

Reply:
@victornoel actually, no, let's leave it as it is. I confirmed in [`tax#in_debt?`](https://github.com/zold-io/zold/blob/de24951af760c7003b5ea96077f20c597d3c9399/lib/zold/tax.rb#L74) that the check is whether `debt > 1 Zold`.

The theory and ruby implementation are not yet stable; it's unfortunately one of the known risks for this java implementation (check the README at the bottom.

Reply:
@llorllale ok, also, I agree that we shouldn't always follow the ruby implementation and try to interpret things by ourselves to refine the theory :)